### ISSUE TITLE
For ImageNet dataset, how to use test daset to test the model? 

### ISSUE THREAD

**Issue by wakeelephant** [jiecaoyu/XNOR-Net-PyTorch#81]
int the article ,only use train and val dataset, two have labels, but for test dataset, there has no labels, that is to say, has not classified, so do not use DataLoader, so how can we use test ? thanks.  @jiecaoyu

**Comment by jiecaoyu** [comment]
We usually only test the validation dataset. Accuracy numbers reported in most research papers is for the validation dataset.

**Comment by wakeelephant** [comment]
ok.Thanks. @jiecaoyu
